packages/components/src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,78 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type ReactElement } from 'react';
+
+export enum DividerStyle {
+    NoLine = 'noline',
+    Dashed = 'dashed',
+    Solid = 'solid',
+    Dotted = 'dotted',
+}
+
+export enum DividerHeight {
+    Small = '36px',
+    Medium = '60px',
+    Large = '96px',
+}
+
+export type DividerProps = {
+    style?: DividerStyle;
+    height?: DividerHeight | string;
+    color?: string;
+    vertical?: boolean;
+    'data-test-id'?: string;
+};
+
+const styleMap = {
+    [DividerStyle.NoLine]: 'tw-border-none',
+    [DividerStyle.Dashed]: 'tw-border-dashed',
+    [DividerStyle.Solid]: 'tw-border-solid',
+    [DividerStyle.Dotted]: 'tw-border-dotted',
+};
+
+const DIVIDER_TEST_ID = 'fondue-divider';
+
+export const Divider = ({
+    vertical = false,
+    style = DividerStyle.Solid,
+    height = DividerHeight.Small,
```

Comment:
Please remove the enum and use union type (`'none' | 'dashed' | 'solid' | 'dotted'`)